.taskcluster.yml

```diff
@@ -2,267 +2,484 @@ version: 1
-              && python tools/taskcluster/release.py \
-                --channel nightly \
-                --commit \
-                --output /opt/focus-android/app/build/outputs/apk \
-                --apk focus/release/app-focus-armeabi-v7a-release-unsigned.apk \
-                --apk focus/release/app-focus-arm64-v8a-release-unsigned.apk \
-                --apk klar/release/app-klar-armeabi-v7a-release-unsigned.apk \
-                --apk klar/release/app-klar-arm64-v8a-release-unsigned.apk
-          artifacts:
-            public/task-graph.json:
-              type: file
-              path: /opt/focus-android/task-graph.json
-              expires: ${expires_in}
-            public/actions.json:
-              type: file
-              path: /opt/focus-android/actions.json
-              expires: ${expires_in}
-            public/parameters.yml:
-              type: file
-              path: /opt/focus-android/parameters.yml
-              expires: ${expires_in}
-        extra:
-          cron: {$json: {$eval: 'cron'}}
-          tasks_for: ${tasks_for}
-        metadata:
-          name: (Focus for Android) Focus/Klar Nightly Builds (Public)
-          description: Decision task scheduled by cron task [${cron.task_id}](https://tools.taskcluster.net/tasks/${cron.task_id})
-          owner: ${event.sender.login}@users.noreply.github.com
-          source: ${repository}/raw/${event.release.tag_name}/.taskcluster.yml
+              $mergeDeep:
+                  # Remove this block when releases have been moved to taskgraph
+                  - $if: 'tasks_for == "github-release" && event["action"] == "published"'
+                    then:
+                      $let:
+                        decision_task_id: {$eval: as_slugid("decision_task")}
+                        expires_in: {$fromNow: '1 year'}
+                        repository: https://github.com/mozilla-mobile/focus-android
+                        scheduler_id: taskcluster-github
+                      in:
+                        taskId: ${decision_task_id}
+                        taskGroupId: ${decision_task_id}  # Must be explicit because of Chain of Trust
+                        created: {$fromNow: ''}
+                        deadline: {$fromNow: '2 hours'}
+                        expires: ${expires_in}
+                        schedulerId: ${scheduler_id}   # Must be explicit because of Chain of Trust
+                        provisionerId: mobile-3
+                        workerType: decision
+                        requires: all-completed   # Must be explicit because of Chain of Trust
+                        priority: highest
+                        retries: 5
+                        scopes:
+                          - assume:repo:github.com/mozilla-mobile/focus-android:release
+                        routes:
+                          - statuses  # Automatically added by taskcluster-github. It must be explicit because of Chain of Trust
```

Comment:
Removing the two lines above may fix the CoT issues. I think we use `checks` instead of `statuses` now.

Reply:
Ben adds `adding https://github.com/mozilla-mobile/fenix/blob/main/.taskcluster.yml#L3 might be good too, regardless`

`verify_cot` was useful in debugging here; it may be a good tool to start using.